priv/repo/migrations/20240425131500_add_basic_permissions.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Trento.Repo.Migrations.AddBasicPermissions do
+  use Ecto.Migration
+
+  def up do
+    execute "INSERT INTO abilities(id, name, resource, label, inserted_at, updated_at) VALUES (1, 'all', 'all', 'Permits all operations on all the resources', NOW(), NOW())"
+
+    execute "INSERT INTO abilities(id, name, resource, label, inserted_at, updated_at) VALUES (2, 'all', 'users', 'Permits all operations on user resource', NOW(), NOW())"
+  end
+
+  def down do
+    execute "DELETE FROM abilities where id = 1"
```

Comment:
I would be careful with `id = 1` kind of comparisons.
I have seen ecto skipping some ids for unknown reasons.
Having something like `DELETE FROM abilities where name = 'all' and resrouce = 'all';` looks more stable, and easier to copy paste for the future. Because otherwise you need to keep of the count of entries, etc

Reply:
It's true, but in this migration we are forcing the sequence so we put id 1 and 2 in the up insert and we are sure of these ids, we can have them hardcoded

Reply:
True :+1: